content/ko/docs/contribute/style/page-content-types.md

```diff
@@ -0,0 +1,218 @@
+---
+title: 페이지 콘텐츠 타입
+content_type: concept
+weight: 80
+---
+
+<!-- overview -->
+
+쿠버네티스 문서는 여러 타입의 페이지 콘텐츠를 따른다.
+
+- 개념
+- 태스크
+- 튜토리얼
+- 레퍼런스
+
+<!-- body -->
+
+## 콘텐츠 섹션
+
+각 페이지 콘텐츠 타입은 마크다운 코멘트와 HTML 헤딩으로 정의된 
+여러 섹션을 포함한다. `heading` 숏코드를 사용하여 페이지에 
+콘텐츠 헤딩을 추가할 수 있다. 코멘트와 헤딩은 페이지 콘텐츠 타입의 구조를 
+유지하는 데 도움이 된다.
+
+페이지 콘텐츠 섹션을 정의하는 마크다운 코멘트의 예시
+
+```markdown
+<!-- overview -->
+```
+
+```markdown
+<!-- body -->
+```
+
+콘텐츠 페이지에 공통적인 헤딩을 생성하려면, 헤딩 문자열과 함께 `heading` 숏코드를 
+사용한다.
+
+헤딩 문자열 예시
+
+- whatsnext
+- prerequisites
+- objectives
+- cleanup
+- synopsis
+- seealso
+- options
+
+예를 들어, `whatsnext` 헤딩을 생성하려면 "whatsnext" 문자열과 함께 heading 숏코드를 추가한다.
+
+```none
+## {{%/* heading "whatsnext" */%}}
+```
+
+`prerequisites` 헤딩은 다음과 같이 선언할 수 있다.
+
+```none
+## {{%/* heading "prerequisites" */%}}
+```
+
+`heading` 숏코드는 하나의 문자열 파라미터를 예상한다.
+헤딩 문자열 파라미터는 `i18n/<lang>/<lang>.toml` 파일의 변수 접두사와 일치한다.
+예시
+
+`i18n/en/en.toml`:
+
+```toml
+[whatsnext_heading]
+other = "What's next"
+```
+
+`i18n/ko/ko.toml`:
+
+```toml
+[whatsnext_heading]
+other = "다음 내용"
+```
+
+## 콘텐츠 타입
+
+각 콘텐츠 타입은 기대되는 페이지 구조를 비공식적으로 정의한다.
+제안된 페이지 섹션으로 페이지 콘텐츠를 생성한다.
+
+### 개념
+
+개념 페이지는 쿠버네티스의 특정 측면을 설명한다. 예를 들어, 개념 페이지는 쿠버네티스 
+디플로이먼트 오브젝트를 설명하고 배포, 스케일링, 업데이트된 애플리케이션에 수행하는 
+역할을 설명할 수 있다. 일반적으로 개념 페이지는 단계별 순서를 포함하지 않고, 
+대신 태스크나 튜토리얼에 대한 링크를 
+제공한다.
+
+새 개념 페이지를 작성하려면, `/content/ko/docs/concepts` 디렉터리의 
+하위 디렉터리에 다음과 같은 특성을 가진 마크다운 파일을 생성한다.
+
+개념 페이지는 세 개의 섹션으로 나뉜다.
+
+| 페이지 섹션      |
+|---------------|
+| overview      |
+| body          |
+| whatsnext     |
+
+`overview`와 `body` 섹션은 개념 페이지에 코멘트로 나타난다.
+`heading` 숏코드를 사용하여 페이지에 `whatsnext` 섹션을 추가할 수 있다.
+
+각 섹션을 콘텐츠로 채우도록 한다. 다음 가이드라인을 따른다.
+
+- H2 및 H3 헤딩으로 콘텐츠를 구성한다.
+- `overview`의 경우, 단일 문단으로 주제의 컨텍스트를 설정한다.
+- `body`의 경우, 개념을 설명한다.
+- `whatsnext`의 경우, 개념에 대해 더 학습할 수 있는 주제의 불릿 리스트(최대 5개)를 제공한다.
+
+[어노테이션](https://kubernetes.io/ko/docs/concepts/overview/working-with-objects/annotations/)은 게시된 개념 페이지의 예시이다.
+
+### 태스크
+
+태스크 페이지는 일반적으로 짧은 단계별 순서를 제공하여 단일 작업을 수행하는 방법을 보여준다. 
+태스크 페이지는 최소한의 설명을 가지지만, 관련 배경과 지식을 제공하는 
+개념 주제에 대한 링크를 종종 제공한다.
```

Comment:
```suggestion
태스크 페이지는 최소한의 간결한 설명을 포함하지만, 관련 배경과 지식을 제공하기 위해서 
종종 개념적인 내용에 대한 링크를 제공한다.
```

A task page shows how to do a single thing, typically by giving a short
sequence of steps. Task pages have minimal explanation, but often provide links
to conceptual topics that provide related background and knowledge.